example_config.yaml

```diff
@@ -1,49 +1,89 @@
 chihaya:
+  # The interval communicated with BitTorrent clients informing them how
+  # frequently they should announce in between client events.
   announce_interval: 15m
-  prometheus_addr: localhost:6880
+
+  # The network interface that will bind to an HTTP endpoint that can be
+  # scraped by an instance of the Prometheus time series database.
+  # For more info see: https://prometheus.io
+  prometheus_addr: 0.0.0.0:6880
+
+  # The maximum number of peers returned in an announce.
   max_numwant: 50
+
+  # The default number of peers returned in an announce.
```

Comment:
Oh, numwant is optional, hmm. You mean something like seeders actually announcing with numwant=0? Problem is: How do we distinguish that from `numwant` missing (=default)? Maybe we should make `numwant` a signed integer again, and set that to -1?